swagger-spec/users/addon_detail.yaml

```diff
@@ -1 +1,51 @@
 # /users/{user_id}/addons/{provider}/
+get:
+  summary: Retrieve an addon
+  description: >-
```

Comment:
I think it would be good to add a "Permissions" section to this description too.